lib/internal/process/task_queues.js

```diff
@@ -143,11 +146,9 @@ function runMicrotask() {
     try {
       callback();
     } catch (error) {
-      // TODO(devsnek): Remove this if
-      // https://bugs.chromium.org/p/v8/issues/detail?id=8326
-      // is resolved such that V8 triggers the fatal exception
-      // handler for microtasks.
-      triggerFatalException(error, false /* fromPromise */);
+      // runInAsyncScope() swallows the error so we need to catch
+      // it and handle it here.
+      triggerUncaughtException(error, false /* fromPromise */);
```

Comment:
@devsnek I tried a cctest on V8 master and it seems the issue still exist. However we still need to catch it anyway since `runInAsyncScope()` will swallow the error.